lib/tasks/colormaps.rake

```diff
@@ -21,9 +33,19 @@ namespace :colormaps do
     capabilities_file = Nokogiri::XML(capabilities_str.sub('xmlns="http://www.opengis.net/wmts/1.0"', ''))
 
     layers = capabilities_file.xpath("/Capabilities/Contents/Layer")
+
     layers.each do |layer|
       id = layer.xpath("./ows:Identifier").first.content.to_s
-      url = layer.xpath("./ows:Metadata/@xlink:href").to_s
+
+      # get v1.2 role metadata node
+      target = layer.xpath("./ows:Metadata[contains(@xlink:role, '1.2')]")
+      if target.empty?
+        # layer does not have a metadata node with xlink:role v1.2,
+        # so try to use url from another metadata node (no version or v1.0)
+        url = layer.xpath("./ows:Metadata/@xlink:href").to_s
```

Comment:
This was the original method to get the url, and is only used if there is no `<ows:Metadata` tag that has a `xlink:role` attribute that ends in `/colormaps/v1.2`. So if the `gibs_url` is an updated version that has the `xlink:role` that has v1.2, it won't use this method, but if the `gibs_url` hasn't been updated to include versions in the role yet it will still pull the urls the old way.